cmake/linux/apprun-hooks/carla-hook.sh

```diff
@@ -1,10 +1,8 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
There are multiple constructs that do not work in every type of shell, e.g. `${0##*/}` will not work in `csh`. What's the idea behind using `/bin/sh`, which could be shell of arbitrary type?